### PR TITLE
NodeLog List

### PR DESCRIPTION
**Pull request by ZobairAlijan**
<h2>Purpose</h2>
Node Log List allows users to be able to get log information.

<h2>New Look</h2>
screen shot 2015-07-24 at 1 44 52 pm

screen shot 2015-07-24 at 1 45 35 pm

### CODE REVIEW

File: limits.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (brianjgeiger) [inline limits.sh]: This file probably should not be checked in.

File: website/static/css/site.css [hunk 2]
```diff
@@ -0,0 +1,2766 @@
+/* Body and structure
```
Comment (brianjgeiger) [inline website/static/css/site.css]: This file definitely should not be checked in.

File: website/static/js/paginator.js [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
+* Paginator model
+*/
 'use strict';
 var ko = require('knockout');
 var oop = require('js/oop');
-var MAX_PAGES_ON_PAGINATOR = 7;
-var MAX_PAGES_ON_PAGINATOR_SIDE = 5;
-
-
+var MAX_PAGES_ON_PAGINATOR = 17;
```
Comment (brianjgeiger) [inline website/static/js/paginator.js]: This is not part of this PR.

File: website/static/js/tests/paginator.test.js [hunk 4]
```diff
@@ -37,8 +37,8 @@ describe('Paginator', () => {
     });
 
     it('previousPage', () => {
-        numberOfPages = 5;
-        currentPage = 3;
+        numberOfPages = 18;
```
Comment (brianjgeiger) [inline website/static/js/tests/paginator.test.js]: This, also, is not part of this PR.

File: website/templates/log_list.mako [hunk 5]
```diff
@@ -13,73 +13,64 @@
     %endif
         id="logScope">
 
+    <div class="logs addon-widget-container">
```
Comment (brianjgeiger) [inline website/templates/log_list.mako]: Not part of this PR.

**Comment by brianjgeiger** [comment]
Ah, I see. You've been doing all your work in develop. You need to use branches and you definitely need to use gitflow for your work.

**Comment by ZobairAlijan** [comment]
pull conflict with develop
